packages/shared/ReactTypeOfWork.js

```diff
@@ -33,9 +33,9 @@ export const HostRoot = 3; // Root of a host tree. Could be nested inside anothe
 export const HostPortal = 4; // A subtree. Could be an entry point to a different renderer.
 export const HostComponent = 5;
 export const HostText = 6;
-export const CallComponent = 7;
-export const CallHandlerPhase = 8;
-export const ReturnComponent = 9;
+export const CallComponent_UNUSED = 7;
```

Comment:
Why are we leaving these in?

Reply:
Might rename/reuse them for next types we add?

Reply:
Does anything rely on the type of work number? Just delete and collapse?

Reply:
DevTools does.